model/metric.go

```diff
@@ -240,7 +219,7 @@ func IsValidMetricName(n LabelValue) bool {
 		}
 		return utf8.ValidString(string(n))
 	default:
-		panic(fmt.Sprintf("Invalid name validation scheme requested: %s", NameValidationScheme.String()))
+		panic(fmt.Sprintf("Invalid name validation scheme requested: %d", scheme))
```

Comment:
Please use string representation:
```suggestion
		panic(fmt.Sprintf("Invalid name validation scheme requested: %s", scheme))
```